package.json

```diff
@@ -1,20 +1,24 @@
 {
   "name": "soundcloud-images",
   "version": "",
-  "description": "",
+  "description": "FEC",
```

Comment:
No one outside of recent HR grads know what this is. Leave blank, remove, or fill in a couple of sentenes.